src/main/steps/applicant1/connection-summary/content.ts

```diff
@@ -5,28 +5,47 @@ import { CommonContent } from '../../common/common.content';
 
 export const jurisdictionMoreDetailsContent = (
   connections: JurisdictionConnections[] | undefined,
+  isDivorce: boolean,
   isRespondent = false
 ): { connectedToEnglandWales: string; readMore: string } => {
   const resConnection = enContainsHabitualResConnection(connections);
```

Comment:
I think it's worth renaming resConnection now, otherwise it's confusing between that and residualConnection

Reply:
Agree.  Changed